integration_tests/features/run_with_custom_target.feature

```diff
@@ -0,0 +1,67 @@
+Feature: `run` command with a custom profile target
+  Background:
+    Given the project 007_run_with_custom_target
```

Comment:
How is this project different from all the others?

Reply:
Maybe what you can change is the `--profiles-dir profiles/broken` in calls and use a common project